packages/liotad/README.md

```diff
@@ -1,23 +1,34 @@
-#Liota Package Manager
-Liota package manager consists of two parts: (1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and (2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
+## Liota Package Manager
+
+Liota package manager consists of two parts:
+
+(1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and
+
+(2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
 
 Package manager will initialize the data structures and threads when its own module (`package_manager.py`) is imported. Once imported, PackageMessengerThread will start listening on the pipe and PackageThread will load all packages specified in the auto list in a batch.
 
-##PackageThread
+## PackageThread
 
 Currently, supported commands include package action commands and statistical commands.
 
-###Package action commands
+### Package action commands
 
-* **load** package_name sha1_checksum [package_name sha1_checksum] ...
+* **load** package_name <sha1_checksum> ...
```

Comment:
Might be useful to show how multiple packages can be loaded, just ... is not intuitive enough.

Something like:

**load** package_name <sha1_checksum> [(optional) package2_name <sha1_checksum_of_package2>...]

Reply:
Done